doc/cephfs/kernel.rst

```diff
@@ -1,58 +1,95 @@
-====================================
- Mount CephFS with the Kernel Driver
-====================================
+=================================
+ Mount CephFS using Kernel Driver
+=================================
 
-It is recommended to install the ``/sbin/mount.ceph`` kernel mount
-helper if working with the kernel cephfs driver. To mount CephFS with
-the kernel driver you may use the ``mount`` command::
+Pre-requisite
+-------------
+Before mounting CephFS, ensure that the client host (where CephFS will be
+mounted and used) has a copy of the keyring for the user which has CAPS for
+the MDS. Keyring must be present on the host where the Ceph Monitor resides.
 
-	# mkdir /mnt/mycephfs
-	# mount -t ceph :/ /mnt/mycephfs
+#. Copy the Ceph keyring from the MON host to client host at ``/etc/ceph``::
 
-Omitting the monitor addresses will cue ``mount.ceph`` to look them up
-in the local configuration file.  If you know at least one of the monitor
-addresses and ports you can specify them directly in a comma-separated
-list, and the kernel will avoid looking them up::
+    # on client host
+    mkdir /etc/ceph
+    sudo scp {user}@{server-machine}:/etc/ceph/ceph.keyring /etc/ceph/ceph.keyring
 
-        # mount -t ceph 192.168.0.1:6789,192.168.0.2:6789:/ /mnt/mycephfs
+#. Ensure that the keyring has appropriate permissions::
```

Comment:
You're talking about the keyring file but the chmod command refers to the mount point.